security/providers/oidc/src/main/java/io/helidon/security/providers/oidc/OidcFeature.java

```diff
@@ -151,7 +152,8 @@ public final class OidcFeature implements HttpFeature {
     private static final String CODE_PARAM_NAME = "code";
     private static final String STATE_PARAM_NAME = "state";
     private static final String DEFAULT_REDIRECT = "/index.html";
-    private static final JsonReaderFactory JSON = Json.createReaderFactory(Map.of());
+    static final JsonReaderFactory JSON_READER_FACTORY = Json.createReaderFactory(Map.of());
```

Comment:
Wrong order of fields - if you change visibility, they should move above private

Reply:
done